tests/wamr-test-suites/spec-test-script/runtest.py

```diff
@@ -1072,6 +1072,15 @@ def compile_wasm_to_aot(wasm_tempfile, aot_tempfile, runner, opts, r, output = '
     # exception isn't thrown in several cases
     cmd.append("--disable-llvm-lto")
 
+    # Bounds checks disabled by default for 64-bit targets, enable it for qemu,
+    # which may be used for 64-bit targets
```

Comment:
OK, shall we update the comments to make it clear? Somewhat like "Bounds-checks is disabled by default for 64-bit targets, but it may be not available in QEMU with Nuttx, so we enable it here"?

Reply:
OK